app/events/proxy_configs/affecting_object_changed_event.rb

```diff
@@ -8,8 +8,12 @@ def self.create(proxy, object)
       object_type: object.class.name,
       metadata: {
         service_id: proxy.service_id,
-        provider_id: proxy.account.id
+        provider_id: proxy.account&.id
```

Comment:
Won’t the event be invalid in this case?

Reply:
Second time this happens to me in less than a month. First add the nil type check, then define the `.valid?` class method and forget to remove the check. Sorry :/